tools/gulp/constants.ts

```diff
@@ -6,6 +6,10 @@ export const SOURCE_ROOT = join(PROJECT_ROOT, 'src');
 export const DIST_ROOT = join(PROJECT_ROOT, 'dist');
 export const DIST_COMPONENTS_ROOT = join(DIST_ROOT, '@angular/material');
 
+export const SASS_AUTOPREFIXER_OPTIONS = {
+  browsers: ['last 2 versions'],
+  cascade: false,
+};
```

Comment:
If I recall correctly, we need to explicitly exclude IE10 and mobile IE because autoprefixer considers "IE" and "Edge" to be separate browsers, thus IE10 falls within the last two versions. This is necessary to prevent it from adding the outdated ms-flex prefixed rules.